.github/workflows/pythonpublish.yml

```diff
@@ -61,8 +68,8 @@ jobs:
           body: ${{ steps.changelog.outputs.changes }}
           token: ${{ github.token }}
 
-      - name: Publish to PyPI
-        env:
-          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
-          TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
-        run: twine upload dist/*
+      - name: Publish distribution 📦 to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          password: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
Based on https://docs.pypi.org/trusted-publishers/ I bet we can (and should) remove this password field.

Reply:
But it works like this in g2p, so I'll leave it this way for now.